analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -751,7 +751,8 @@ def main(args):
         if not os.path.exists(log_file):
             LOG.error("The specified logfile '%s' does not exist!",
                       log_file)
-            continue
+            sys.exit(1)
```

Comment:
Do we support the case when multiple logfiles are provided? Is this an intentional behavior that the first non-existing logfile will cease the analysis and the remaining logfiles will not be used?

Reply:
We were able to provide multiple log files for the analyze command but in case of this we print an error message and returned with an error code:
https://github.com/Ericsson/codechecker/blob/1bb40031ce750f8b759c27f89bb23edff48f7b7b/analyzer/codechecker_analyzer/cmd/analyze.py#L704-L706
I changed the behaviour of this and we will support only one log file.

Reply:
I like this change, but unfortunately this breaks a few things. Now `args.logfile` is a string instead of a list. Unfortunately all usages should be inspected now to see if this change results an exception somewhere in runtime. Maybe grepping `args.logfile` could be a good starting point.